web/routes.go

```diff
@@ -63,3 +70,17 @@ func initiateTranscriptionJobHandler(w http.ResponseWriter, r *http.Request) {
 func healthHandler(w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte("healthy!"))
 }
+
+var statuses = [...]string{
+	"In progress",
+	"Done",
+	"Error",
+}
+
+func jobStatusHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
I'm all for comments on unexported functions even if they aren't strictly required.